java-spiffe-core/src/main/java/io/spiffe/svid/jwtsvid/JwtSvid.java

```diff
@@ -105,7 +115,9 @@ private JwtSvid(final SpiffeId spiffeId,
      */
     public static JwtSvid parseAndValidate(@NonNull final String token,
```

Comment:
What do you think of keeping a `parseAndValidate` with the current signature to not introduce API breaking changes and add a new one that takes the `String hint`? The former would just call the latter passing null as `hint` argument.